pkg/receiver/mysqlrecordsreceiver/README.md

```diff
@@ -0,0 +1,134 @@
+# MySQL Records Receiver
+
+This receiver queries MySQL for database records and creates a log record (plog.Logs type) for each database record.
+
+Supported pipeline types: logs
+
+BasicAuth Password Encryption Use Case:
+
+- The receiver supports password encryption for 'BasicAuth' authentication_mode
+- To generate an encrypted password:
+  - Specify encrypt_secret_path to a secret file containing a 24 character string
+  - Include the following in the otel config:
+    service:
+      telemetry:
+        logs:
+          level: debug
+    The encrypted password will only be printed in the console with a debug log level. Once generated, the user can remove the telemetry field so as to enable logging at the default info level. To use the encrypted password, the user needs to specify password_type as 'encrypted' and also the encrypt_secret_path to the same secret file.
```

Comment:
Flow is weird. I need to run OTC to generate password and then I need to run it again in order to use it?

Reply:
For the first iteration of the run, the password will be generated. It will fetch database queries too if required, but the encrypted password won't be used. The encrypted password can be used from the second iteration if required via the user passing the password in the config file.

Reply:
Is it possible to generate password externally?